test/scenarios/17. settings/advanced/show-test-networks.csv

```diff
@@ -0,0 +1,12 @@
+Advanced settings: Check “Show test networks” toggle works correctly,,,,
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots) and click on the Settings from menu.,,Settings page appears,
+4,Click on the Advanced tab.,,Advanced settings page appears,
+5,Validate that the 'Show test networks' is default to off.,,The toggle is off by default,
+6,Toggle the 'Show test networks' switch.,,The switch is turned on as expected,
+7,Click on the 'Ethereum Mainnet' drop down on the left top,,Select a network dialog box appears,
+8,Validate that the 'Show test networks' toggle is selected by default.,,The 'Show test networks' toggle is selected by default,
+9,"Validate that the test networks - 'Goerli',  'Sepolia' and 'Linea Goerli' appear",,The test networks is shown as expected,
```

Comment:
Dencun fork will be the last before Goerli is [deprecated](https://blog.ethereum.org/2023/11/30/goerli-lts-update).  To help minimize maintenence requirements, maybe we should consider NOT listing out each displayed testnet by name, and rather confirm that all expected testnets are presented.  That said, off the top of my head Im not sure of the best source of truth to validate expectations at the time of the test.

Reply:
ok i could make it generic, just testnet and not mention any names.